qng/cli_qng.py

```diff
@@ -0,0 +1,94 @@
+import argparse
+import json
+import operator
+import os
+import random
+import sys
+import unicodedata
+from typing import Optional
+
+import qng
+from qng import common
```

Comment:
As a general rule of thumb I prefer doing e.g. `import qng.common` over `from qng import common` to preserve namespacing. Not a huge deal though.

Reply:
But you do `from typing import Optional` just above :wink:.

Reply:
Yes, I loosely follow the [Google Python style guide](http://google.github.io/styleguide/pyguide.html#22-imports) which has an explicit exemption for the `typing` module. I think the rationale is that otherwise it would make function signatures much too verbose.